tools/configen/configen/templates/module.j2

```diff
@@ -10,9 +10,9 @@ from dataclasses import dataclass, field
 @dataclass
 class {{class_name}}Conf:
     _target_: str = "{{ classes_map[class_name].target }}"
-{% if classes_map[class_name].convert %}
-    _convert_: 'all'
-{% endif %}
+{% for flag in classes_map[class_name].default_flags %}
+    {{ flag.name }}: {{ flag.type_str }} = "{{ flag.default }}"
```

Comment:
flags can be boolean as well. (like _recursive_).
I think this should be implemented by adding the flags to the parameters like other parameters.